packages/forms-web-app/src/dynamic-forms/questions.js

```diff
@@ -1602,6 +1602,7 @@ exports.questions = {
 		html: 'resources/your-details/applicant-name.html',
 		fieldName: 'applicantName',
 		url: 'applicant-name',
+		formatType: 'contactDetails',
```

Comment:
can we move the 3 uses of this string into a const
maybe in multi-field-input/question.js

Reply:
No need for this now, I've set a formatJoinString field on the InputField which handles formatting - default is for each input to appear on a new line.